packages/project-access/src/index.ts

```diff
@@ -14,7 +14,8 @@ export {
     isCapNodeJsProject,
     loadModuleFromProject,
     readCapServiceMetadataEdmx,
-    readUi5Yaml
+    readUi5Yaml,
+    getMtaPath
```

Comment:
`getMtaPath` is also not inserted in alphabetic order :(.

Reply:
Updated in alphabetic order